functions.py

```diff
@@ -0,0 +1,73 @@
+#1
+description = ('Country', [
+    '2011 ', '2012 ', '2013 ', '2014 ', '2015 ', '2016 ', '2017 ', '2018 ',
+    '2019 '])
+raw_data = [
+    ('AL', [': ', ': ', ': ', ': ', ': ', ': ', ': ', '84 ', ': ']),
+    ('AT', ['75 ', '79 ', '81 ', '81 ', '82 ', '85 ', '89 ', '89 ', '90 ']),
+    ('BA', [': ', ': ', ': ', ': ', ': ', ': ', ': ', '69 ', '72 ']),
+    ('BE', ['77 ', '78 ', '80 ', '83 ', '82 ', '85 ', '86 ', '87 ', '90 ']),
+    ('BG', ['45 ', '51 ', '54 ', '57 ', '59 ', '64 ', '67 ', '72 ', '75 ']),
+]
+countries = {
+    "AL" : "Albania" ,
+    "AT" : "Austria" ,
+    'BA' : 'Bosnia and Herzegovina' ,
+    'BE' : 'Belgium' ,
+    'BG' : 'Bulgaria'
+    }
+result = {
+    'Romania': [
+                {'year': '2019', 'coverage': 84},
+                {'year': '2018', 'coverage': 67},
+    ],
+    'Germany': [
+                {'year': '2019', 'coverage': 84},
+                {'year': '2018', 'coverage': 67},
+    ],
+}
+#dictionare_mici = dic_year_cov(YYYY, raw_data[1])
+years_clear = [
+        int(elem.strip(' '))
+        for elem in description[1]
+    ]
+#print('Clear years are: YEARS_CLEAR')
+#print(years_clear)
+
+#gasim lista coresponzatoare tarii raw_data[(tara[lista_cu_cov._data_points])]
+def get_coverage_cc(cc):

```

Comment:
I don't see those small functions used but I guess are good for implementing the two functions for year data and country data. I like how they are written